cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/io/cdap/cdap/etl/lineage/LineageOperationsProcessor.java

```diff
@@ -86,15 +87,17 @@ public LineageOperationsProcessor(Set<Connection> stageConnections,
 
   private Map<String, Operation> computeProcessedOperations() {
     Map<String, Operation> processedOperations = new HashMap<>();
+    // this stores information about all the outputs on a merge stage, the key is the prefix of the merge
+    Map<String, List<String>> mergedOutputs = new HashMap<>();
```

Comment:
I don't understand what this represents. What is a merge stage? What is a prefix of a merge?

Reply:
Removed this to reduce complication